Samples/SimpleViewer/SimpleViewer.cs

```diff
@@ -176,11 +172,40 @@ private void backgroundWorker_DoWork(object sender, DoWorkEventArgs e)
                     saveSnapshot = false;
                 }
                 this.BeginInvokeEx(f => pictureBox.Image = bmp);
+
+                fps++;
+                DateTime now = DateTime.Now;
+                if(now - lastSecond > new TimeSpan(0, 0, 1))
+                {
+                    int fpsCopy = fps;
+                    this.BeginInvokeEx(f => labelFps.Text = $"{fpsCopy} fps");
+                    lastSecond = now;
+                    fps = 0;
+                }
+
             }
             DisconnectCamera();
             isBgwFinished.Set();
         }
 
+        private static void TrimDepthImage(FloatCameraImage img)
+        {
+            float minDepth = Properties.Settings.Default.MinDepthToDisplay;
+            float maxDepth = Properties.Settings.Default.MaxDepthToDisplay;
```

Comment:
I would pass `minDepth` and `maxDepth` as parameters to `TrimDepthImage`. Settings should be read by the application to allow reusing the method.

I would also remove "depth" from method and parameter names. Nothing about it is specific to depth images.

Reply:
Done.